README.md

```diff
@@ -25,13 +25,14 @@
 
 </div>
 
-- [**Website**](#website)
-- [**Scope**](#scope)
-- [**Getting Started**](#getting-started)
-- [**Contributing**](#contributing)
-- [**Feedback**](#feedback)
-- [**Roadmap**](#roadmap)
-- [**License**](#license)
+- [Website](#website)
```

Comment:
Looks like the VS Code MD extension reformatted this automatically. Will roll-back